src/store/selectors/decals.js

```diff
@@ -31,7 +31,7 @@ export const selectDecalsByUserId = createSelector(
   [selectDecals, getUserId],
   (decals, userId) => {
     return Object.values(decals).filter((decal) => {
-      return decal.attributes && decal.attributes.userId === userId
+      return decal?.attributes?.userId === userId
```

Comment:
uhh I'm gonna accept this as-is for now, but these null checks should no longer be required after the decal state object stops tracking if the user is eligible.

Reply:
Yeah, it simply so it'll load. Definitely should be revisted.